huaweicloud/resource_huaweicloud_fgs_function_v2_test.go

```diff
@@ -26,13 +27,17 @@ func TestAccFgsV2Function_basic(t *testing.T) {
 				Config: testAccFgsV2Function_basic(rName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckFgsV2FunctionExists(resourceName, &f),
+					resource.TestMatchResourceAttr(resourceName, "urn", regexp.MustCompile(`\w+`)),     // just check urn is not null
```

Comment:
we should use `TestCheckResourceAttrSet` to check the attribute is not null.

Reply:
changed to use `TestCheckResourceAttrSet `